packages/compass-import-export/src/csv/csv-utils.ts

```diff
@@ -192,11 +192,20 @@ const parenthesis = {
 };
 
 function isEJSON(value: string) {
-  return (
+  if (
     value.length &&
     ['{', '['].includes(value[0]) &&
     value[value.length - 1] === parenthesis[value[0] as '{' | '[']
-  );
+  ) {
+    try {
+      JSON.parse(value);
```

Comment:
drive-by. @Anemy originally suggested this and we discussed it before and I was on the fence about it. But I just noticed that one of my fixtures happens to be full of false positives so I tried this with it and even though it is a fairly large file it didn't noticeably slow it down, so now I'm convinced.